app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,17 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title Forex
```

Comment:
I think in this project we will use only english language, but still all phrases should be defined in 'locales' files in the same way as in 'Motohelper'.

Reply:
Ok I'll add this locale files in new task.